yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -400,8 +414,6 @@ private[spark] class Client(
     // same name but different path files are added multiple time, YARN will fail to launch
     // containers for the app with an internal error.
     val distributedNames = new HashSet[String]
-    YarnSparkHadoopUtil.get.obtainTokenForHiveMetastore(sparkConf, hadoopConf, credentials)
-    YarnSparkHadoopUtil.get.obtainTokenForHBase(sparkConf, hadoopConf, credentials)
     if (credentials != null) {
```

Comment:
this is now a bit out of place, the idea was to dump the tokens for debugging after they were filled.  We should move this after the obtainCredentials call above.